packages/fiori/src/ShellBar.ts

```diff
@@ -459,16 +521,37 @@ class ShellBar extends UI5Element {
 	@slot()
 	midContent!: Array<HTMLElement>;
 
+	/**
+	 * Define the items displayed in the start of the additional content area.
+	 * @public
+	 * @since 2.5.0
+	 */
+	@slot({ type: HTMLElement, individualSlots: true })
+	additionalContextStart!: Array<HTMLElement>;
```

Comment:
lets rename this to startContent / endContent to be align with other component slot names